Description: Call help2man with --no-discard-stderr to avoid build failures
Author: Alessio Treglia <alessio@debian.org>
Forwarded: no
Last-Update: 2011-05-05

--- a/man/Makefile.am
+++ b/man/Makefile.am
@@ -9,4 +9,4 @@
 # binary when any source file changes, only the main one.
 # Use -o so that the `missing' program can infer the output file.
 toonloop.1: $(top_srcdir)/src/application.cpp $(common_mandeps) $(top_srcdir)/man/man-toonloop.txt
-	$(HELP2MAN) --include=man-toonloop.txt --name="The Toonloop Live Animation Performance Tool" --no-info $(top_builddir)/src/toonloop -o $@
+	$(HELP2MAN) --include=man-toonloop.txt --name="The Toonloop Live Animation Performance Tool" --no-info --no-discard-stderr $(top_builddir)/src/toonloop -o $@
